localflavor/bg/fields.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+from .validators import egn_validator, eik_validator
+
+
+class EGNField(models.CharField):
```

Comment:
That was my intention but it's hard to read with all the capital letters. Is there a way to expand `EGN` and `EIK` that will make sense?